invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/customFields/CustomField.js

```diff
@@ -0,0 +1,68 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2020-2022 CERN.
+// Copyright (C) 2020 Northwestern University.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import _get from "lodash/get";
+import _set from "lodash/set";
+import _cloneDeep from "lodash/cloneDeep";
+import _isArray from "lodash/isArray";
+import { Field } from "../serializers";
+
+export class CustomField extends Field {
+  constructor({
+    fieldpath,
+    deserializedDefault = null,
+    serializedDefault = null,
+    allowEmpty = false,
+    vocabularyFields = [],
+  }) {
+    super({ fieldpath, deserializedDefault, serializedDefault, allowEmpty });
+    this.vocabularyFields = vocabularyFields;
+  }
+
+  #mapCustomFields(record, customFields, mapValue) {
```

Comment:
Not added in this PR but the name starting with `#` triggered my curiosity.

Reply:
its a private class attr (if that is what you are asking about): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields